usagov/usagov-2021
# USAGOV-1499-Analytics-Heading

Moved the logo outside of header and added alt text and new target


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1499

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Easiest way to test this is using "Go Live" extension in vscode.
- From website-analytics/index.html use Go Live to open preview site.
- No data will be loaded that is fine for this ticket.
- First make sure the H1 text (Analytics Dashboard) behaves like a link (should take you back to "analytics dashboard home" http://127.0.0.1:8080/website-analytics/ ) 
- Second make sure the logo also behaves like a link (this one should take you to usa.gov) 

Test all the pages: 
-- On the drop down chose another value and select Go.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
